.devcontainer/postCreateCommand.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+
+sudo apt update && sudo apt install -y --no-install-recommends \
+    clang
```

Comment:
This should also probably clone the GNU coreutils as it takes a reasonable amount of time to clone, and a git fetch would likely be much faster.

The expectation is that the repo ends up one level up from this workspace in /workspace/gnu

```shell
git clone --recurse-submodules https://github.com/coreutils/coreutils.git "/workspaces/gnu"
```
This either requires sudo or root as /workspace is 644 root:root. So it's likely easiest to run as root - unsure if that's a problem within a container. Obviously you wouldn't want this on a physical machine